### PR TITLE
fix CVE-2023-2976 and CVE-2023-34462

### PR DESCRIPTION
**Pull request by alanlvle**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
You need to update change logs and licenses to adopt these dependencies lib changes.
And tests codes don't matter.

**Comment by alanlvle** [comment]
update

**Comment by wu-sheng** [comment]
I can't see the guava version update in the license file.

File: apm-sniffer/apm-sdk-plugin/guava-eventbus-plugin/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <artifactId>apm-guava-eventbus-plugin</artifactId>
 
     <properties>
-        <guava.version>31.0.1-jre</guava.version>
+        <guava.version>32.0.1-jre</guava.version>
```
Comment (lujiajing1126) [inline apm-sniffer/apm-sdk-plugin/guava-eventbus-plugin/pom.xml:30]: This is not necessary since it should be a provided dependency

**Review by lujiajing1126 — CHANGES_REQUESTED** [review]

**Comment by wu-sheng** [comment]
All versions in test are not necessary, actually.

**Comment by alanlvle** [comment]
> All versions in test are not necessary, actually.

I need revert it?

**Comment by lujiajing1126** [comment]
> > All versions in test are not necessary, actually.
> 
> I need revert it?

Sure. Pls only make necessary changes.

**Comment by wu-sheng** [comment]
It's fine

**Review by wu-sheng — DISMISSED** [review]
I am good in either way. Rely on @wallezhang

**Review by lujiajing1126 — APPROVED** [review]
lgtm

**Pull request merged**